deepcell/utils/train_utils.py

```diff
@@ -54,16 +54,26 @@ def get_callbacks(model_path,
         tensorboard_log_dir (str): log directory for tensorboard.
         monitor (str): quantity to monitor.
         verbose (int): verbosity mode, 0 or 1.
+        manual_save (int or None): optional parameter specifying period
+            to save model at, irrespective of validation accuracy improvements
 
     Returns:
         list: a list of callbacks to be passed to model.fit()
     """
-    cbs = [
-        callbacks.ModelCheckpoint(
-            model_path, monitor=monitor,
-            save_best_only=True, verbose=verbose,
-            save_weights_only=save_weights_only),
-    ]
+    if manual_save is None:
+        cbs = [
+            callbacks.ModelCheckpoint(
+                model_path + ".h5", monitor=monitor,
+                save_best_only=True, verbose=verbose,
+                save_weights_only=save_weights_only),
+        ]
+    else:
+        cbs = [
+            callbacks.ModelCheckpoint(
+                model_path + "_epoch_{epoch:02d}.h5", period=manual_save,
```

Comment:
maybe we can generalize the model name with epoch somehow?

Reply:
I was wondering about that. We could save the epoch as part of the file name every time the model saves, but that could start taking up a lot of space